ReactSkia/BUILD.gn

```diff
@@ -37,6 +37,9 @@ source_set("ReactSkia") {
     "components/RSkComponentText.h",
     "components/RSkComponentView.cpp",
     "components/RSkComponentView.h",
+    "views/view/RSkLayoutHelper.h",
+    "views/view/RSkLayoutHelper.cpp",
+
```

Comment:
is new folder "views" required here? Since this helper functions will be used by components , will it need to be part of components ?